tests/lib_test.py

```diff
@@ -255,20 +255,19 @@ def test_no_files(self) -> None:
             gather_symbols_provided(file_path)
 
     def test_app9(self) -> None:
-        file_path = self._get_test_path("app_9.py")
+        file_path = self._get_test_path("app_9_test.py")
         result = gather_symbols_provided(file_path)
         assert "report" in result
         assert str(file_path) in result["report"]
         assert set(result["report"][str(file_path)]) == {
-            "tests.data.app_9.A",
-            "tests.data.app_9.B",
-            "tests.data.app_9.GLOBAL_VAL",
-            "tests.data.app_9.SomeClass",
-            "tests.data.app_9.X",
-            "tests.data.app_9.Y",
-            "tests.data.app_9.async_signal_handler",
-            "tests.data.app_9.b",
```

Comment:
@fridex this line was failing `pytest`, so I deleted, WDYT, is it OK?

Reply:
This looks suspicious. Why was that? Was it failing even before?

Reply:
Looks like fixed now.